pkg/storage/impl/mysql/mysql_impl.go

```diff
@@ -37,12 +63,103 @@ func (impl *MySQLImpl) GetConnectionType() taxonomy.ConnectionType {
 	return impl.Name
 }
 
-// storage allocation - placeholder
+func dsn(username, password, hostname, port, dbName string) string {
+	return fmt.Sprintf("%s:%s@tcp(%s:%s)/%s", username, password, hostname, port, dbName)
```

Comment:
Printing password is problematic

Reply:
Password is not printed. Here it's just formatting a string.